testschematic/test_options.go

```diff
@@ -189,6 +192,8 @@ func TestSchematicOptionsDefault(originalOptions *TestSchematicOptions) *TestSch
 		newOptions.SchematicsApiURL = DefaultSchematicsApiURL
 	}
 
+	newOptions.SkipTestTearDown = false
```

Comment:
should only default to false if the value is not set in originalOptions (if it is set to true).

In golang, the default for a bool is already false, so setting this here is probably optional.